lib/js/helpers/event.js

```diff
@@ -0,0 +1,80 @@
+// ┌──────────────────────┐
+// │ DOM Event Management │
+// └──────────────────────┘
+
+// returns standard interaction event, later will add touch support
+export function click () {
+  return 'click';
+}
+
+// add a callback function to an event on a DOM node
+export function add (domNode, e, fn) {
+  if (domNode.addEventListener) {
+    return domNode.addEventListener(e, fn, false);
+  } else if (domNode.attachEvent) {
+    return domNode.attachEvent('on' + e, fn);
+  }
+}
+
+// remove a specific function binding from a DOM node event
+export function remove (domNode, e, fn) {
+  if (domNode.removeEventListener) {
+    return domNode.removeEventListener(e, fn, false);
+  } else if (domNode.detachEvent) {
+    return domNode.detachEvent('on' + e,  fn);
+  }
+}
+
+// get the target element of an event
+export function target (e) {
+  return e.target || e.srcElement;
+}
+
+// prevent default behavior of an event
+export function preventDefault (e) {
+  if (e.preventDefault) {
+    return e.preventDefault();
+  } else if (e.returnValue) {
+    e.returnValue = false;
+  }
+}
+
+// stop and event from bubbling up the DOM tree
+export function stopPropagation (e) {
+  e = e || window.event;
+  if (e.stopPropagation) {
+    return e.stopPropagation();
+  }
+  if (e.cancelBubble) {
+    e.cancelBubble = true;
+  }
+}
+
+// return a funciton that will only execute
```

Comment:
this has probably always been there, but `function` is spelled wrong in this comment...